modules/network-observability-total-resource-usage.adoc

```diff
@@ -21,26 +20,15 @@ Metrics exported to Prometheus can impact the resource usage. Cardinality values
 .Total average resource usage
 [%autowidth, options="header"]
 |===
-| Sampling value | Parameters | Test 1 (25 nodes) | Test 2 (65 nodes) | Test 3 (120 nodes)
-.6+| *Sampling = 1* | *With Loki* 3+|
-| Total NetObserv CPU Usage | 3.24 | 3.42 | 7.30
-| Total NetObserv RSS (Memory) Usage | 14.09 GB | 22.56 GB | 36.46 GB
-| *Without Loki* 3+|
-| Total NetObserv CPU Usage | 2.40 | 2.43 | 5.59
-| Total NetObserv RSS (Memory) Usage | 6.85 GB | 10.39 GB | 13.92 GB
-.6+| *Sampling = 50* | *With Loki* 3+|
-| Total NetObserv CPU Usage | 2.04 | 2.36 | 3.31
-| Total NetObserv RSS (Memory) Usage | 8.79 GB | 19.14 GB | 21.07 GB
-| *Without Loki* 3+|
-| Total NetObserv CPU Usage | 1.55 | 1.64 | 2.70
-| Total NetObserv RSS (Memory) Usage | 6.71 GB | 10.15 GB | 14.82 GB
-.6+| *Sampling = 400* | *With Loki* 3+|
-| Total NetObserv CPU Usage | 1.71 | 1.44 | 2.36
-| Total NetObserv RSS (Memory) Usage | 8.21 GB | 16.02 GB | 17.44 GB
-| *Without Loki* 3+|
-| Total NetObserv CPU Usage | 1.31 | 1.06 | 1.83
-| Total NetObserv RSS (Memory) Usage | 7.01 GB | 10.70 GB | 13.26 GB
+| Sampling value | Resources used | Test 1 (25 nodes) | Test 2 (250 nodes)
+.3+| *Sampling = 50*
+| Total NetObserv CPU Usage | 1.35 | 5.39
+| CPU per node              | 0.054 | 0.022
+| Total NetObserv RSS (Memory) Usage | 16 GB | 63 GB
+.3+| *Sampling = 1*
+| Total NetObserv CPU Usage | 1.82 | 11.99
+| CPU per node              | 0.072| 0.048
+| Total NetObserv RSS (Memory) Usage | 22 GB | 87 GB
```

Comment:
@memodi How come there is no "RSS (Memory) per node"?

@memodi The math doesn't add up.
 0.022 cpu/node * 250 nodes = 5.5 (not 5.39)
  
 Let's fix the rounding issue.
 0.072 cpu/node * 25 nodes = 1.8 (says 1.82) 

Use "Network Observability" instead of "NetObserv".

Reply:
@stleerh we do have per pod data if we need. Here it seems it's been just divided by the number of nodes and being rounded off. 

While I was initially okay with the idea of mentioning it per node basis when @jotak had proposed, now I am having some doubts, since the resource consumption is not evenly distributed across the nodes as it's been suggested when we mention per node usage. For example the infra/master nodes will have higher usages as they have heavy workloads running on them as compared to other worker nodes.

Reply:
> One of the most important numbers is what percentage of resources (CPU and memory) is Network Observability taking with respect to the resources used by the cluster. In other words,
> 
> NOO resource usage / Cluster resource usage (with or without NOO)
> 
> We are missing the CPU and memory cluster usage numbers.
> 
> Note: If we want the numbers to look and sound better, "Cluster resource usage" is the total traffic including NOO 😊.

@stleerh we do have those numbers as well, see: 

https://docs.google.com/spreadsheets/d/1QCMchUgl3_9p1Q55uEklmNmtpUWapuygQEJCEJKl0Wk/edit?gid=1278573333#gid=1278573333&range=20:23

https://docs.google.com/spreadsheets/d/1jUQeoWXCGvPoU1WqAvwnLvU5l5fA-xGXGxHQ_lqyRd0/edit?gid=1284697683#gid=1284697683&range=20:23

Reply:
> so I need to remove the CPU per node

I'd be inclined to that, but I'd like others weigh in.

> add TotalClusterCPUUsage and TotalClusterRSSUsage from those sheets you link instead? 

@stleerh can share some thoughts on how that could be presented in the docs.

Reply:
Aren't there just 3 control plane nodes compared to 25 or 250 worker nodes?  In any case, I'm okay with removing the CPU per node.  I would naively assume you divide by the number of workers to get this data.  More importantly, I would like to know the percentage of resource usage rather than the number of cores or RAM used.

Reply:
@stleerh is the "percentage of resource usage" the NetObservClusterCPUPercentUtil from the sheet?

Reply:
As discussed in SCRUM, we'll table the discussion for new data points to add to the table for another iteration as it seems this discussion maybe needs more consideration and there's lots of differing opinions. For the sake of this update, we are keeping mostly the same kinds of information we had in the previous table, just pared down so its not as much of a maintenance burden for QE perf run time.

Reply:
I will remove CPU per node